### ISSUE TITLE
Issue loading a large amount of scans (pointclouds) in the same scene

### ISSUE THREAD

**Issue by worlderland** [potree/potree#1159]
Hello everyone,

We are experiencing an issue when trying to load multiple pointclouds in the same scene (1000+ scans) from a MS Azure VM (VM is extremely powerful, in fact we have absolutely no issues with a single, giant octree).

We based ourselves on the example "multiple_pointclouds.html" in the Potree examples folder, and we are basically looping through a long list of paths where each metedata.json file sits and load each octree.
What we are experiencing is the crash of the browser (Chrome) while trying to load such a great amount of files.

In conclusion, we were wondering whether there is anyone who has suggestions about loading multiple clouds in the same scene, as well as performance advices to have it done in the best (and fastest) way possible.

Thank you in advance.

**Comment by HeadLessHUN** [comment]
You need to do dynamic loading. Load only that you require currently, unload the unnecessary octrees. You should only load 4-5 octree at a time.
Convert an overview cloud, and then load the high detailed ones if required.

**Comment by paujar** [comment]
Is there an example how to implement that kind of overview and to smaller LODs?
